File: mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height;
    int spaces;

    // taking input from user
    do
    {
        height = get_int("Height: ");
    }
    while (height < 1 || height > 8);

    // printing the pattern
    for (int i = 0; i < height; i++)
    {
        spaces = height - 1 - i;
        for (int y = 0; y < height; y++)
        {
            if (y < spaces)
            {
                printf(" ");
            }
            else
            {
                printf("#");
            }
        }
        printf("\n");
    }
}

File: sentimental-mario-more/mario.py
def inputNum():
    try:
        num = int(input("Height: "))  # Taking users input
        if num > 0 and num < 9:
            spaces = num - 1
            while spaces >= 0:  # Loop to print the pattern
                print(" " * spaces, end="")
                print("#" * (num - spaces), end="")
                print(" " * 2, end="")
                print("#" * (num - spaces), end="")
                print("")
                spaces -= 1
        else:
            inputNum()
    except:
        inputNum()


inputNum()

